.github/actions/lint/action.yml

```diff
@@ -0,0 +1,77 @@
+name: Lint Scripts, Styles, and Texts
+description: Lint JavaScript, Sass, CSS, and Markdown files
+
+inputs:
+  scripts:
+    required: false
+    default: ''
+    description: |
+      Information about the projects where JavaScript linting should occur. Can be in two formats:
+      - A comma separated list of relative paths to the project directories. These are the paths where a package.json
+        file could be, if desired. If there is no package.json on the provided path, the default one from this
+        repository is copied and the default asset subpath (./Assets/Scripts) is assumed. Using this approach, you have
+        to copy the assets to their final path (in ./wwwroot) using a different method, for example using the Copy
```

Comment:
Add an example of the Copy task (just a copy of what you previously added) to the Readme with the simple config sample.

Reply:
Educating the reader on basic MSBuild features? Feels like mission creep.

Reply:
We tell the user here what to do, but leave figuring out the details as an exercise to the reader. Not really an exercise that's worth doing every single time.